app/assets/stylesheets/sass/views/employees/_index.scss

```diff
@@ -72,3 +72,66 @@
 .fa-solid {
   margin: 5px;
 }
+
+/* The switch - the box around the slider */
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+}
+
+/* Hide default HTML checkbox */
+.switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
```

Comment:
Sort the items.

Also, you can take advantage of the nesting property that SASS has.

Reply:
Fixed!